### ISSUE TITLE
document edge cluster setup

### ISSUE THREAD

**Issue by pdgetrf** [CentaurusInfra/fornax#45]
### Goal 

this task is to document edge cluster setup procedures, including:

- [x] 1-layer edge cluster
- [x] multiple-layer edge cluster
- [ ] mission deployment through multiple-layer
  - mission in the right state
  - deployment in the right state
  - pod in the right state 
- [x] mission status check
- [x] edge cluster status check
- [ ] verify [k3s](https://k3s.io/) cluster on the edge (no sub-cluster from k3s cluster)
- [ ] edge AI application (stretch goal)

### Deliverable 

A set of markdown docs/wiki

### Time estimate
~ 2 weeks by 17th

**Comment by davidzhu22** [comment]
https://github.com/davidzhu22/fornax/blob/setupguide/docs/setup-guide/cluster_setup.md

**Comment by davidzhu22** [comment]
https://github.com/davidzhu22/fornax/blob/setupguide/docs/setup-guide/vm_setup.md

**Comment by davidzhu22** [comment]
https://github.com/davidzhu22/fornax/blob/setupguide/docs/setup-guide/edge_cluster_copy.md

**Comment by pdgetrf** [comment]
Hi David, once the setup document is ready for review, please file a PR against the fornax repo to check it in.

Document can be continuously worked on so it doesn't have to be perfect. Let's get this checked with the best condition we can get by this week.

**Comment by pdgetrf** [comment]
moving this task to the "review" phase for final review and fixup.
